ETL.md

```diff
@@ -0,0 +1,144 @@
+# ETL
+
+The ETL is a high level structure within Kubecost tasked with, as it name suggest, extracting cost and usage data from various sources (Prometheus, Thanos, Cloud Service Providers), transforming it into Asset and Allocation objects and loading it into caches that allow for fast querying.
+
+## Cloud Stores
+The ETL contains a Map of Cloud Stores, each of which represent an integration with a Cloud Service Provider. Each Cloud Store is responsible for the Cloud Asset and Reconciliation Pipelines which add Out-of-Cluster costs and Adjust Kubecost's estimated cost respectively via cost and usage data pulled from the Cloud Service Provider. Each Cloud Store has a unique identifier called the `ProviderKey` which varies depending on which Cloud Service Provider is being connected to and ensures that duplicate configurations are not introduced into the ETL. 
+
+- AWS: Account Id of the Master Billing Account
```

Comment:
On AWS, it's possible to link just a sub-account... in case that conflicts with ProviderID implementation

Reply:
Yeah it is just whatever billing account the cur was, I made this a little more general